pyuvdata/miriad.py

```diff
@@ -410,8 +425,8 @@ def read_miriad(self, filepath, correct_lat_lon=True, run_check=True, run_check_
             warnings.warn(str(ve))
 
         # check if object has all required uv_properties set
-        if run_check:
-            self.check(run_check_acceptability=run_check_acceptability)
+#        if run_check:
+#            self.check(run_check_acceptability=run_check_acceptability)
```

Comment:
Whoops.... Yes, I cut those from mine because it was running slow and didn't want to address it, but I'll put them back and run the profiler again.

I don't think it'll break, since write_miriad also runs the acceptability check.